Audiobook Player/PlayerViewController.swift

```diff
@@ -171,132 +171,136 @@ class PlayerViewController: UIViewController {
 
     @objc func sliderChanged(_ sender: UISlider) {
         let percentage = sender.value / sender.maximumValue
-        
+
         if let audioPlayer = PlayerManager.sharedInstance.audioPlayer {
             audioPlayer.currentTime = TimeInterval(percentage) * audioPlayer.duration
         }
     }
-    
+
     @IBAction func presentMore(_ sender: UIButton) {
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
-        let vc = storyboard.instantiateViewController(withIdentifier: "MoreViewController") as! MoreViewController
-        
-        self.presentModal(vc, animated: true, completion: nil)
+
+        if let viewController = storyboard.instantiateViewController(withIdentifier: "MoreViewController") as? MoreViewController {
+            self.presentModal(viewController, animated: true, completion: nil)
+        }
     }
-    
+
     @IBAction func presentChapter(_ sender: UIButton) {
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
-        let vc = storyboard.instantiateViewController(withIdentifier: "ChaptersViewController") as! ChaptersViewController
-        
-        self.presentModal(vc, animated: true, completion: nil)
+
+        if let viewController = storyboard.instantiateViewController(withIdentifier: "ChaptersViewController") as? ChaptersViewController {
+            self.presentModal(viewController, animated: true, completion: nil)
+        }
     }
-    
+
     @IBAction func presentSpeed(_ sender: UIButton) {
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
-        let vc = storyboard.instantiateViewController(withIdentifier: "SpeedViewController") as! SpeedViewController
-        
-        self.presentModal(vc, animated: true, completion: nil)
+
+        if let viewController = storyboard.instantiateViewController(withIdentifier: "SpeedViewController") as? SpeedViewController {
+            self.presentModal(viewController, animated: true, completion: nil)
+        }
     }
-    
-    
-    @IBAction func didSelectChapter(_ segue:UIStoryboardSegue){
-        
+
+    @IBAction func didSelectChapter(_ segue: UIStoryboardSegue) {
+
         guard PlayerManager.sharedInstance.isLoaded() else {
             return
         }
-        
-        let vc = segue.source as! ChaptersViewController
-        let chapter = vc.currentChapter!
-        
-        PlayerManager.sharedInstance.setChapter(chapter)
+
+        if let viewController = segue.source as? ChaptersViewController {
+            let chapter = viewController.currentChapter!
+
+            PlayerManager.sharedInstance.setChapter(chapter)
+        }
     }
-    
-    @IBAction func didSelectSpeed(_ segue:UIStoryboardSegue){
-        
+
+    @IBAction func didSelectSpeed(_ segue: UIStoryboardSegue) {
+
         guard PlayerManager.sharedInstance.isLoaded() else {
```

Comment:
in this guard, let's add the check of `let viewController = segue.source as? SpeedViewController`